conference.js

```diff
@@ -1202,11 +1198,14 @@ export default {
             items[key] = param[1];
         }
 
-        this.e2eeKey = items.e2eekey;
+        if (typeof items.e2eekey !== undefined) {
+            APP.store.dispatch(setE2EEKey(items.e2eekey));
 
-        logger.debug(`New E2EE key: ${this.e2eeKey}`);
+            // Clean URL in browser history.
+            const cleanUrl = window.location.href.split('#')[0];
```

Comment:
Don't we have the clean url in redux?

Reply:
Note that thiss is only executed if and only iff the e2ekey thing triggers the onhashchange, so I'd say yeah.

Reply:
But still. See https://github.com/jitsi/jitsi-meet/pull/6087#discussion_r409714242 .